src/client_side_reply.h

```diff
@@ -150,6 +150,12 @@ class clientReplyContext : public RefCountable, public StoreClient
     } CollapsedRevalidation;
 
     CollapsedRevalidation collapsedRevalidation;
+
+    /// HTTP response body bytes stored in our Client Stream buffer (if any)
+    StoreIOBuffer lastStreamBufferedBytes;
+
+    // TODO: Remove after moving the meat of this function into a method.
+    friend CSR clientGetMoreData;
```

Comment:
I guess this can be removed if another request (about inlining requestMoreBodyFromStore()) is addressed).